bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties

```diff
@@ -0,0 +1,9 @@
+bin.includes = META-INF/,\
+               OSGI-INF/,\

```

Comment:
This folder does not exist, which makes the Maven build fail - so this line should be removed.